qctrlopencontrols/qiskit/quantum_circuit.py

```diff
@@ -165,8 +180,8 @@ def convert_dds_to_qiskit_quantum_circuit(
 
         if offset_distance < 0:
             raise ArgumentsValueError(
-                "Offsets cannot be placed properly. Spacing between the rotations"
-                "is smaller than the time required to perform the rotation. Provide"
+                "Offsets cannot be placed properly. Spacing between the rotations "
```

Comment:
I'd remove all these unrelated changes from the PR -- keep it focused on the lazy loading.

Reply:
Ah did not mean to do that; reverted all these cosmetic changes.